core/store/migrate/migrations/0259_add_workflow_artifacts.sql

```diff
@@ -0,0 +1,43 @@
+-- +goose Up
+-- +goose StatementBegin
+-- Create the workflow_artifacts table
+CREATE TABLE workflow_artifacts (
```

Comment:
Might as well call this workflow_secrets since it'll only be used to store secrets